install.sh

```diff
@@ -655,6 +655,14 @@ openrc_start() {
   $SUDO $FILE_NODE_EXPORTER_SERVICE restart
 }
 
+# relabel to executable if SElinux is installed
+selinux() {
+  if getenforce > /dev/null; then
```

Comment:
This review is on an earlier version of the PR. Please update.
Also, please read about the reason why I use `type` instead of `command` for SELinux. https://github.com/carlocorradini/node_exporter_installer/pull/18#issuecomment-1239439112